getting stared ios

corinnekrych opened the pull request:
@cvasilak @lfryc here is a getting started with iOS, mind to review?

cvasilak commented on docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc:
```diff
@@ -0,0 +1,52 @@
+- Target the latest release: install Yosemite, Xcode 6.1.1. iOS 8.1.3
+
+== Online Training
+If you like screencast and MOOC session, I'd recommand the excellent Stanford classes. You can find them for free on link:https://itunes.apple.com/us/course/developing-ios-8-apps-swift/id961180099[iTunes].
```
s/recommand/recommend

cvasilak commented on docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc:
```diff
@@ -0,0 +1,52 @@
+If you like screencast and MOOC session, I'd recommand the excellent Stanford classes. You can find them for free on link:https://itunes.apple.com/us/course/developing-ios-8-apps-swift/id961180099[iTunes].
+
+== Books
+There are plenty of good books to learn from, among goog one here is a book with tutorials from link:http://www.raywenderlich.com/store/ios-apprentice[Ray Wenderlich].
```
s/goog/good

cvasilak commented on docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc:
```diff
@@ -0,0 +1,52 @@
+Runtime for Swift based demo and libraries is 8.X.
+
+==== AeroGear 1.6.X
+Our earlier version work from Xcode 5 and iOS 6. 
```
s/from/for

cvasilak commented on docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc:
```diff
@@ -0,0 +1,52 @@
+==== AeroGear 1.6.X
+Our earlier version work from Xcode 5 and iOS 6. 
+
+**Note** Althgouh still supported AeroGear 1.6.X is a deprecated branch, all new features goes in 2.X version (on master for all libraries).
```
s/Althgouh/Although

cvasilak replied:
s/goes/go

cvasilak commented on docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc:
```diff
@@ -0,0 +1,52 @@
+=== Build with cocoapods
+
+Our Swift and ObjC libraries are build using Cocoapods 0.36.0.
+In your Podfile add the AeroGear dependecy you're intersted in (here for example OAuth2):
```
s/intersted/

cvasilak commented on docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc:
````diff
@@ -0,0 +1,52 @@
+In your Podfile add the AeroGear dependecy you're intersted in (here for example OAuth2):
+
+```bash
+pod 'AeroGearOAuth2'
````
I think we should we add a note for ```use_frameworks!``` required flag

lfryc commented:
:heart: love this guide

perhaps we could add a link guiding people to `/ios/` landing page at the bottom?

corinnekrych commented:
@cvasilak sorry for type (I should have used a syntax corrector :/) i fixed those + adding a note on use_frameworks!
@lfryc I've added the link to ios main page

lfryc commented:
:+1: Awesome, thanks @corinnekrych , merging

lfryc commented:
:rocket: landed: https://github.com/aerogear/aerogear.org/commit/d4f6f1b449241a431b88ce8db31696bf4509f7a4
